### PR TITLE
example usage

### LINKED ISSUE #12: Add usage example

### PR DESCRIPTION
**Pull request by agga1**
closes #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,32 @@ pacman -S libfdk-aac
 ```
 brew install fdk-aac
 ```
+## Sample usage
```
Comment (mickel8) [inline README.md:39]:
```suggestion
## Usage
```

File: README.md [hunk 2]
```diff
@@ -36,7 +36,32 @@ pacman -S libfdk-aac
...
 
+```elixir
```
Comment (mickel8) [inline README.md:41]: A second example for Decoder would be useful too. 
Also, let's add short description what this example does. Something like 
```
The following pipeline takes wav file as input and encodes it as AAC. 
```

**Review by mickel8 — CHANGES_REQUESTED** [review]

**Comment by mickel8** [comment]
One more thing, could we add missing badges to README? You can take their format from `membrane_template_plugin`.

File: README.md (hunk 2 above)
Comment (agga1) [reply README.md:46]: Done.

File: README.md (hunk 1 above)
Comment (agga1) [reply README.md]: Done.

**Review by mickel8 — APPROVED** [review]

**Pull request merged**